src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -572,7 +573,7 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
   selectTextTrack(textTrack: PKTextTrack): void {
     const textTracks = this._videoElement.textTracks;
     if ((textTrack instanceof PKTextTrack)
-      && (textTrack.kind === 'subtitles' || textTrack.kind === 'captions')
+      && (textTrack.kind === TRACK_KIND.SUBTITLES || textTrack.kind === TRACK_KIND.CAPTIONS)
```

Comment:
you already do `[TRACK_KIND.CAPTIONS, TRACK_KIND.SUBTITLES].includes(textTracks.kind)` a few lines below, so let's use it here as well.